cli2/morphir-make2.ts

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
```

Comment:
Can you remove the `2` from the name of the command? We already have this inside `cli2` so no need to differentiate further.